prow/config.yaml

```diff
@@ -2489,6 +2489,31 @@ presubmits:
         hostPath:
           path: /mnt/disks/ssd0
 
+  jlewi/mlkube.io:
```

Comment:
hummm, I feel like our bot might not have access to your repo, you might want to configure somewhere around https://github.com/kubernetes/test-infra/blob/master/prow/plugins.yaml#L5-L11

Reply:
also https://github.com/kubernetes/test-infra/blob/master/prow/plugins.yaml#L37-L38

Reply:
Thanks. I also think I need to configure webhooks on my repo but I'll do that after this PR is done.